spec/api/commentable_spec.rb

```diff
@@ -32,6 +32,11 @@ def thread_result(commentable_id, params={})
         threads.index{|c| c["body"] == "can anyone help me?"}.should_not be_nil
         threads.index{|c| c["body"] == "it is unsolvable"}.should_not be_nil
       end
+      it "returns standalone threads if explicitly requested" do
+        threads = thread_result "question_1", context: "standalone"
```

Comment:
Yeah, that was what I was asking. If that's the case, @benmcmorran , why are we passing in the context here? It shouldn't be necessary, should it?